releases/release-4.0.6.md

```diff
@@ -0,0 +1,130 @@
+---
+title: TiDB 4.0.6 Release Notes
+---
+
+# TiDB 4.0.6 Release Notes
+
+Release date: September 9, 2020
+
+TiDB version: 4.0.6
+
+## Improvements
+
++ TiDB
+
+    - Replace error code and message with standard error [#19888](https://github.com/pingcap/tidb/pull/19888)
+    - Table: improve the write performance of partition table [#19649](https://github.com/pingcap/tidb/pull/19649)
+    - Optimize the performance of encoding plan. [#19279](https://github.com/pingcap/tidb/pull/19279)
+    - Record more RPC runtime information in cop runtime stats [#19264](https://github.com/pingcap/tidb/pull/19264)
+
++ TiFlash
+
+    - Add some metrics about apply Region snapshots and ingest SST files [#1088](https://github.com/pingcap/tics/pull/1088)
+    - Fix the problem that CN check doesn't work on https_port and metrics_port [#1064](https://github.com/pingcap/tics/pull/1064)
+
```

Comment:
+ PD

    - Dashboard: update to v2020.09.08.1 [#2928](https://github.com/pingcap/pd/pull/2928)
    - Change back to the original way to control the low space threshold [#2875](https://github.com/pingcap/pd/pull/2875)
    - Support standard error code [#2918](https://github.com/tikv/pd/pull/2918) [#2911](https://github.com/tikv/pd/pull/2911) [#2913](https://github.com/tikv/pd/pull/2913) [#2915](https://github.com/tikv/pd/pull/2915) [#2912](https://github.com/tikv/pd/pull/2912) [#2907](https://github.com/tikv/pd/pull/2907) [#2906](https://github.com/tikv/pd/pull/2906) [#2903](https://github.com/tikv/pd/pull/2903) [#2806](https://github.com/tikv/pd/pull/2806) [#2900](https://github.com/tikv/pd/pull/2900) [#2902](https://github.com/tikv/pd/pull/2902)

Reply:
Updated.